### ISSUE TITLE
How to config the two phys

### ISSUE THREAD

**Issue by fxj000** [ehntoo/100base-t1-converter#1]
hi, ehntoo,
How are you!
I am very happy to see you put this a good project. and I already draw a board, very similar with yours.
I used TJA1100 and LAN8710. but I don't know how to config the two PHYs, or do not need to config them any more? but so far it does not work. I am very sad.
This is my email: fxj460@163.com.  please kind to give me a hand. thank you very much. 
Please reach out me and I can send my project file to you. Thank you.

**Comment by ehntoo** [comment]
Hi, sorry to have missed this until now. The PHYs are both configured with strap resistors - with the TJA1100 (or TJA1101) placed into reverse MII mode, there should (theoretically) be no need to do any additional configuration over MII. Check out Table 10 in the TJA1100 datasheet for the necessary straps on that chip: https://www.nxp.com/docs/en/data-sheet/TJA1100.pdf

For the TJA1101 used here, I gathered the same information from a smattering of different schematics I found.

Good luck with your project!
